index-test.js

```diff
@@ -37,4 +37,35 @@ test("form submits expected result", () => {
   result.textContent = "";
 });
 
-*/
\ No newline at end of file
+*/
+
+setTimeout(() => {
+
+  test("deleteItem() function should remove an item from the list", () => {
```

Comment:
Yaaaaaaay 🎉 🥳  we have more tests for our binge list project, great work here ✨ 

very neat 😄